vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -58,6 +58,9 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
     // This counter keeps increasing even after the limit is reached.
     private _trackAjaxAttempts: number = 0;
 
+    // array with max length of 2 that store current url and previous url for SPA page route change trackPageview use.
+    private _urlArr: string[] = [];
```

Comment:
Does this need to be an array? Can it instead be just 2 variables?

Reply:
yes~ update with 2 variables